genesyscloud/util/test_utils.go

```diff
@@ -374,3 +374,18 @@ func RandString(length int) string {
 func interfaceToString(val interface{}) string {
 	return fmt.Sprintf("%v", val)
 }
+
+func AssignRegion() string {
+
+	var (
+		region string
+	)
+
+	if v := os.Getenv("GENESYSCLOUD_REGION"); v == "tca" {
+		region = "us-east-1"
+	} else if v == "us-east-1" {
+		region = "us-west-2"
+	}
+	regionJSON := "[" + strconv.Quote(region) + "]"
+	return regionJSON
+}
```

Comment:
If it is possible for the value in `GENESYSCLOUD_REGION` to not be "tca" or "us-east-1" , then this function will return `[""]` 

Should we maybe return a default region in this case?

Reply:
modified it to return default region